signingworker/data/signing_task_schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "title": "Taskcluster MAR signing task minimal schema",
+    "type": "object",
+    "properties": {
+        "scopes": {
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "type": "string"
+            }
+        },
+        "payload": {
+            "type": "object",
+            "properties": {
+                "signingManifest": {
+                    "type": "string"
+                }
+            },
```

Comment:
As discussed elsewhere, it would be nice to get rid of the signing manifest and put its contents right in the payload at some point. The indirection here makes it more difficult to debug. Not a blocking issue, but it would be really nice to have it fixed.